Orange/widgets/data/owdatasets.py

```diff
@@ -539,8 +518,15 @@ def closeEvent(self, event):
         self.header_state = bytes(self.view.header().saveState())
         super().closeEvent(event)
 
-    def load_data(self, path):  # pylint: disable=no-self-use
-        return Orange.data.Table(path)
+    def load_and_output(self, path):  # pylint: disable=no-self-use
+        if path is None:
+            self.Outputs.data.send(None)
+        else:
+            data = Orange.data.Table(path)
```

Comment:
The previous `load_data` method is overriden in Single Cell Datasets.
You can still call it here and make this a new method instead of changing it from `load_data`, which should keep everything working downstream.